core/dbt/task/run.py

```diff
@@ -125,25 +125,24 @@ def run_hooks(self, adapter, hook_type: RunHookType, extra_context):
 
             hook_text = '{}.{}.{}'.format(hook.package_name, hook_type,
                                           hook.index)
-            print_hook_start_line(hook_text, idx, num_hooks)
-
             hook_meta_ctx = HookMetadata(hook, self.index_offset(idx))
-            uid_ctx = UniqueID(hook.unique_id)
-            with uid_ctx, hook_meta_ctx, startctx:
-                logger.debug('on-run-hook starting')
-            status = 'OK'
+            running_ctx = DbtModelState({'node_status': 'running'})
+            with UniqueID(hook.unique_id):
+                with hook_meta_ctx, startctx, running_ctx:
```

Comment:
We actually don't want to use the `running_ctx` here - I had to check how we do this for other "regular" types of nodes - that status is only emitted at the end of the resource's execution. Can we just remove the `running_ctx` from this line (and the definition above)?

Sorry for the confusion there! This one was my bad.

Reply:
actually i'll just push a fix for this haha